wordpress-mobile/gutenberg-mobile
# Fix android html mode scrolling

Fix: https://github.com/wordpress-mobile/gutenberg-mobile/issues/768

This PR separates the ui part of html editor for Android and iOS because for Android we need to wrap TextInput with a ScrollView, on the other hand on iOS this causes the following issue https://github.com/wordpress-mobile/gutenberg-mobile/issues/744 

**To Test**

WPAndroid:
- Switch to HTML mode
- Verify html content is scrolling with the title
- Verify scrolling is fast and smooth
- Verify when we edit title and html content and switch back to visual mode we see the edits

WPiOS:

Before starting:
Update podfile gutenberg commit with the last commit of this PR because there's a new bridge method that had been merged to develop which is unrelated with this PR, but WPiOS podfile wasn't updated. Make sure you run rake dependencies and clean build folder beforehands.

- Switch to HTML mode
- Verify html content is scrolling in its own bounds
- Verify scrolling is fast and smooth
- Verify when we edit title and html content and switch back to visual mode we see the edits
